### ISSUE TITLE
Bug: settings dialog opens another instance of itself on close on mobile 

### ISSUE THREAD

**Issue by monacodelisa** [AnguHashBlog/angular-primeng#54]
### Describe the bug

While working on initial sitewide responsiveness, and making adjustments to the dialogs, 
I saw that when I close the settings dialog it immediately creates another instance of itself which 
is also not aligned well.
- chrome simulator 
- mobile device 

### Steps to reproduce

1. View either `posts` or `series`
2. click on the gear icon to open up the settings dialog
3. close the settings dialog
4. observe the new one appearing 

![image](https://github.com/AnguHashBlog/angular-primeng/assets/64324417/0d6e5da5-abfe-4668-8b9d-1a61411a163b)

### Browsers

Chrome

### I would be willing to submit a PR to fix this issue

- [X] Yes and I have read the [Contribution Guide](https://anguhashblog.com/contributing) and I agree to follow this project's [Code of Conduct](https://anguhashblog.com/code-of-conduct)
- [ ] No

**Comment by MestiriBaha** [comment]
I've been trying to resolve this bug, trying many approaches. Even after inspecting and debugging the code, I'm still not convinced of the cause of this abnormal behavior. It's so weird that this issue only occurs in the mobile version! It could be due to the media query styles of the mobile version , What do you think might be causing this tricky bug ?

**Comment by monacodelisa** [comment]
@MestiriBaha yes this is a very peculiar issue indeed. It happens only on mobile which is the first odd thing, but also - it is the only modal that does that, there are more modals in the app and as far as i remember the rest did not have this behavior  🤔
